csrc/device_lower/lower2device.cpp

```diff
@@ -346,6 +346,9 @@ IdModelOptions getIdModelOptions(Fusion* fusion) {
       if (ldst->opType() == LoadStoreOpType::CpAsyncBulkTensorTile ||
           ldst->opType() == LoadStoreOpType::CpAsyncBulk) {
         options.setBuildTensorIndexer(true);
+        if (ldst->opType() == LoadStoreOpType::CpAsyncBulk) {
```

Comment:
Should there be a check for `1D` TMA here?

Reply:
`LoadStoreOpType::CpAsyncBulk` is `1D`, `nD` uses `LoadStoreOpType::CpAsyncBulkTensorTile`